scientific_library/tvb/simulator/plot/time_series_plotter.py

```diff
@@ -434,17 +432,15 @@ def plot_trajectories(self, time_series, subplots=None, special_idx=[], subtitle
 
     @staticmethod
     def plot_tvb_time_series_interactive(time_series, first_n=-1, **kwargs):
-        from tvb.simulator.plot.timeseries_interactive import TimeSeriesInteractive
-        interactive_plotter = TimeSeriesInteractive(time_series=time_series, first_n=first_n)
+        from tvb.simulator.plot.timeseries_interactive import TimeSeriesInteractivePlotter
```

Comment:
Can this import be put at the top of the file, or it generated circular dependencies ?